src/components/Article.jsx

```diff
@@ -1 +1,46 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = (props) => {
```

Comment:
Task says that we take only 3 arguments as props, so you can use decomposition in your function arguments

Reply:
Yeah, but checklist says that it`s also good